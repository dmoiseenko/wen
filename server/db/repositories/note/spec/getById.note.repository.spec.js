const bootstrap = require('../../../bootstrap/bootstrap.js');
const getNoteById = require('../getById.note.repository.js');


beforeEach(() => bootstrap());

it('should query user by email', async () => {
  const user = await getNoteById(1);

  expect(user).toMatchSnapshot();
});
