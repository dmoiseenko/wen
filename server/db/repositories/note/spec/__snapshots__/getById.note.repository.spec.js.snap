// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should query user by email 1`] = `
Object {
  "id": 1,
  "plain": Object {
    "id": 1,
    "text": "Well done is better than well said. Benjamin Franklin",
  },
  "text": "Well done is better than well said. Benjamin Franklin",
}
`;
